src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -179,24 +186,47 @@ public async Task<bool> UpdateAsync(string userId, string roleId)
             return true;
         }
 
+        private List<UserReleaseRole> GetUserReleaseRoles(string userId)
+        {
+            var result = new List<UserReleaseRole>();
+
+            result.AddRange(_contentDbContext.UserReleaseRoles
```

Comment:
I'm not sure what the point in result.AddRange here is to an empty list, rather than just returning the result of the query?

Reply:
Fixed it